libraries/ms-helper/inc/exported_enums.h

```diff
@@ -24,3 +24,9 @@ typedef enum {
   EE_CHARGER_CONN_STATE_CONNECTED,
   NUM_EE_CHARGER_CONN_STATES,
 } EEChargerConnState;
+
+typedef enum {
```

Comment:
I think it makes sense just to have a general relay open/close. I've renamed it in my motor controller PR.

Reply:
Done.